tests/unittest.py

```diff
@@ -140,7 +140,7 @@ def assertObjectHasAttributes(self, attrs, obj):
             try:
                 self.assertEquals(attrs[key], getattr(obj, key))
             except AssertionError as e:
-                raise (type(e))(e.message + " for '.%s'" % key)
+                raise (type(e))("Assert error for '.{}':".format(key)) from e
```

Comment:
Why was this one updated to not use `e.message`? It looks like this is going to lose information, or are you hoping that by doing the `from e` it will be kept?

Reply:
`e.message` is deprecated and shouldn't be used (which is kinda the point of `B306`), with `from e`, it retains that message still in the trace back (under "caused by"), all this intermediate exception is doing is insert the `key` variable for extra information.

Reply:
From your comment above I think the answer is that this is not a `DependencyException`, so `e.message` is actually deprecated.